src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString() {
+        return "Ball color = " + color + ", "
+                + "Ball number = " + number;

```

Comment:
I'd suggest using the default format of `toString()` to avoid mentioning `Ball` each time
```suggestion
        return "Ball{" 
                + "color=" + color
                + ", number=" + number
                + "}";

```